src/lib.rs

```diff
@@ -34,13 +34,17 @@
 //! [`examples/bot`]: https://gitlab.com/whatisaphone/rlbot-rust/blob/master/examples/bot/main.rs
 //! [`examples/simple`]: https://gitlab.com/whatisaphone/rlbot-rust/blob/master/examples/simple.rs
 
+#![deny(warnings, missing_docs)]
```

Comment:
Ahh yeah I hadn't seen the pre-commit stuff when I put this in. I'll go ahead and remove it. For both warnings and missing docs?

Reply:
I am hoping that if we have `warn(missing_docs)` in the module, and `-D warnings` in the rustc call, it will fail both, only on CI. If `warn(missing_docs)` doesn't work like I'm hoping, then let's remove them both.